server/services/v1/query_runner.go

```diff
@@ -307,6 +307,9 @@ func (runner *InsertQueryRunner) Run(ctx context.Context, tx transaction.Tx, ten
 	if err != nil {
 		return nil, ctx, err
 	}
+	if coll.Type() != api.CollectionType_DOCUMENTS {
```

Comment:
Let' create a function for it?
if err := checkCollType(api.CollectionType_DOCUMENTS); err != nil {
   return err
}
or at least define error variable so we don't duplicate it message.

Reply:
I'll do first, easy to return the custom message with the method name.